pyschema/contrib/__init__.py

```diff
@@ -0,0 +1,8 @@
+import warnings
+
+warnings.warn(
+    "pyschema.contrib is deprecated and will be removed.\n"
+    "Please use the pyschema_extensions package instead.",
+    FutureWarning,
```

Comment:
Yeah, it's so annoying that `DeprecationWarning` won't show by default... I intend to revert this patch pretty soon anyway, as soon as we have switched to the open source build and successfully migrated away from the deprecated api.

Maybe I should use DeprecationWarning and run warnings.simplefilter('default') to re-enable them instead?